### PR TITLE
moved cards around and moved google maps to faq section

### PR DESCRIPTION
**Pull request by Manan-dev**
google-map

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -768,6 +750,34 @@ <h4>I don't know what to build — any ideas?</h4>
...
+                 
+                <div class="card">
+                    <h4>When and where is VolHacks?</h4>
+                    <p>VolHacks will take place during Fall 2021 (Oct. 29th - 31st), at the University of Tennessee,
+                        Knoxville in the Min Kao EECS Building located at 
+                            <a target="_blank" 
+                                href="https://www.google.com/maps/place/Department+of+Electrical+Engineering+and+Computer+Science/@35.958735,-83.924121,18z/data=!4m5!3m4!1s0x0:0x5ae47667cdb72706!8m2!3d35.9586585!4d-83.9245267?hl=en-US">
+                                1520 Middle Dr, Knoxville, TN 37996.</a></p>
+                                <div class="google-maps">
+                                    <iframe  src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d1614.7594968387325!2d-83.924121!3d35.958735!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x5ae47667cdb72706!2sMin%20Kao%20Building%20-%20Department%20of%20Electrical%20Engineering%20%26%20Computer%20Science!5e0!3m2!1sen!2sus!4v1625249092770!5m2!1sen!2sus" width="475" height="125" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
```
Comment (andreicozma1) [inline index.html:772]: For width can we use 100% and/or move it to CSS instead of inline?

File: index.html [hunk 2]
```diff
@@ -310,7 +310,7 @@ <h3 class="table-header">Friday, October 29th</h3>
             <tr>
                 <td class="left">Team Formation</td>
                 <td class="middle">Dough 416</td>
-                <td class="right">7 pm</td>
+                <td class="right">7 Pm</td>
```
Comment (andreicozma1) [inline index.html:313]: Should we keep fully capitalized (like PM) or fully lowercase (pm) @hunterprice04 @Manan-dev

**Review by andreicozma1 — CHANGES_REQUESTED** [review]
See comments above

**Review by andreicozma1 — APPROVED** [review]
Looks good! Good to merge whenever

**Review by hunterprice04 — APPROVED** [review]
great job

**Pull request merged**